code/__HELPERS/type2type.dm

```diff
@@ -424,4 +424,23 @@ proc/tg_text2list(text, glue=",", assocglue=";")
 		if(temp <= 16)
 			. = 0
 		else
-			. = max(0, min(255, 138.5177312231 * log(temp - 10) - 305.0447927307))
\ No newline at end of file
+			. = max(0, min(255, 138.5177312231 * log(temp - 10) - 305.0447927307))
+
+//Argument: Give this a space-separated string consisting of 6 numbers. Returns null if you don't
+/proc/text2matrix(var/matrixtext)

```

Comment:
Further investigation reveals that BYOND only uses the `matrix()` proc for transform matrices, and uses `list()`s for color matricies. So this is correct for it's purpose.